Drop-In/src/main/java/com/braintreepayments/api/dropin/utils/PaymentMethodType.java

```diff
@@ -11,7 +11,7 @@
 import androidx.annotation.Nullable;
 
 public enum PaymentMethodType {
-
+    // `getFrontResource` is pulling icons from android-card-form, `R.drawable` icons are drop-in internal
```

Comment:
❤️ thank you for adding this comment, will definitely help us the next time we have to update icons